### PR TITLE
[onert] Add quant8 support for Mul cker

### PR DESCRIPTION
**Pull request by hasw7569**
Add quantization support for Mul cker

ONE-DCO-1.0-Signed-off-by: Leehansol <hansolu.lee@samsung.com>

### CODE REVIEW

**Comment by hasw7569** [comment]
This PR must be merged after [#2345](https://github.com/Samsung/ONE/pull/2345) merged.

File: compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h [hunk 1]
```diff
@@ -380,6 +429,76 @@ inline void MulSimpleBroadcast(int size, const BinaryArithmeticOpParam &params,
...
+    for (int i0 = 0; i0 < y0; ++i0)
+    {
+      const uint8_t *input2_data_ptr = nullptr;
+      for (int i1 = 0; i1 < y1; ++i1)
+      {
+        input2_data_ptr = input2_data_reset;
+        for (int i2 = 0; i2 < y2; ++i2)
+        {
+          // The input may be switched here, but the common parameters here
+          // do not matter as they will not influence the float math execution.
```
Comment (dr-venkman) [inline compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h:490]: maybe change the `float` in comment to int.

File: compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h [hunk 2]
```diff
@@ -450,6 +569,27 @@ inline void BroadcastMulFivefold(const BinaryArithmeticOpParam &params,
...
+                                       const Shape &output_shape, uint8_t *output_data)
+{
+  if (params.broadcast_category == BroadcastableOpCategory::kGenericBroadcast)
+  {
+    const std::function<uint8_t(const BinaryArithmeticOpParam &, const uint8_t &, const uint8_t &)>
+        fn = [](const BinaryArithmeticOpParam &params, const uint8_t &a,
+                const uint8_t &b) -> uint8_t {
+      return static_cast<uint8_t>(quant8_mul(params, a, b));
+    };
+    reference::BroadcastBinaryArithmeticOpSlowQuant8(params, input1_shape, input1_data,
```
Comment (dr-venkman) [inline compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h:584]: This function call has a dependency to pull request #2345 , hence the build errors.

**Comment by hasw7569** [comment]
PTAL @hseok-oh

**Comment by hseok-oh** [comment]
Please remove `[draft]` header if ready.

File: runtime/onert/backend/cpu/ShapeFixer.cc [hunk 3]
```diff
@@ -64,17 +64,7 @@ void ShapeFixer::visit(const ir::operation::Sub &node)
...
-  // The same as Add
-  const auto lhs_index{node.getInputs().at(ir::operation::Mul::Input::LHS)};
-
-  // Quantization : not supported
-  if (_ctx.at(lhs_index).typeInfo().type() == ir::DataType::QUANT_UINT8_ASYMM)
-  {
-    throw std::runtime_error{"ShapeFixer: NYI for quantized Mul"};
-  }
-}
+void ShapeFixer::visit(const ir::operation::Mul &) { /* DO NOTHING */}
```
Comment (hseok-oh) [inline runtime/onert/backend/cpu/ShapeFixer.cc:67]: You can remove this function if do nothing.

Comment (hasw7569) [reply runtime/onert/backend/cpu/ShapeFixer.cc]: removed visit() function.
Thanks for your comment : )

**Pull request merged**